src/VerifyHandler.php

```diff
@@ -26,6 +27,27 @@
 
 class VerifyHandler implements VerifyHandlerInterface
 {
+    /**
+     * Dependency injection configuration
+     *
+     * @config
+     * @var array
+     */
+    private static $dependencies = [
+        'Logger' => LoggerInterface::class . '.mfa',
+    ];
+
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger = null;
+
+    public function setLogger(LoggerInterface $logger): self
```

Comment:
SS's Injector can provide null for this property, unfortunately with strict typing we need to be loose:

```suggestion
    public function setLogger(?LoggerInterface $logger): self
```

Reply:
Sorry, it's the `getLogger()` equivalent that can return null, you're not using one so ignore this